src/routes/votesRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+const { updateVote } = require("../controllers/votesController");
+
+// @route   PUT /api/v1/votes/:activityID
+router.put("/:activityID", updateVote);
```

Comment:
Sorry to be pedantic about API design again, but since we are creating a `vote` now, I think the request should be `POST /votes` and the activity id should be included in the post data. (Eventually, user id would be included in the post data as well.)

Reply:
The vote resource is created automatically once a new activity resource is added. So from my understanding, it makes sense for this request to be a `PUT` since the vote tally associated with `activityID` is updated. In the current implementation a middleware is set in the Activity model to automatically add a new vote resource whenever a new activity is created, so no activity resource will be without a vote resource.

Reply:
> from my understanding, it makes sense for this request to be a PUT since the vote tally associated with activityID is updated

Okay, I see how you're thinking about it. I guess following this logic, the `id` in the url should be the vote id. But again, if you'd like to just move this along and we can make edits/tweaks when we add users, I'm good with that. 👍